css/style.css

```diff
@@ -0,0 +1,72 @@
+body {
+  font-family: Roboto;
+  font-weight: 900;
+}
+
+p {
+  letter-spacing: 2.4px;
+}
+
+.container {
+  margin: 0 auto;
+  text-align: center;
+}
+
+header > .container {
```

Comment:
plz don't rewrite styles with the current selector, each selector should have unique styles